pipelines/coref/requirements.txt

```diff
@@ -0,0 +1,3 @@
+spacy-transformers>=1.1.6,<1.2.0
+https://github.com/explosion/spacy-models/releases/download/en_core_web_lg-3.3.0/en_core_web_lg-3.3.0-py3-none-any.whl
+git+https://github.com/explosion/spacy-experimental@feature/coref#egg=spacy-experimental
```

Comment:
Does the readme tell users to install these requirements? As installing them is not part of a workflow?

Reply:
Also I think this is not quite right. If you have a custom install of `spacy`, this will pull in the latest install from `pypi` instead :(

Reply:
I think we should updat all of this to be compatible with 3.4 & the new models, as people trying out experimental features are likely to have the latest `spacy` version,  IMO

Reply:
I think there are several unresolved questions around how to handle this that are bigger than this PR. 

1. **Do we tell users to install this, or do we automate it?** I think it makes sense to either tell users to `pip install -r requirements.txt` or assume they will know to do it anyway. In some projects we had the project do this (or similar) for the user, but we can't automate everything because projects don't run without spaCy. #121 is related. In particular I think it makes sense to have the user be partly responsible for this because I don't think there's a good way to automate GPU library installation. 
2. **spaCy 3.3.** I was actually testing this on 3.4 but lowered it for this PR due to the version pin in `spacy-experimental`. The pin has just been raised in the PR there so I can raise it here as well.

> Also I think this is not quite right. If you have a custom install of spacy, this will pull in 3.3.1 from pypi instead :(

I did not realize that would happen, I'll need to see which part of the requirements is doing that. Or we can do something like just explicitly state in the README "please install spaCy with GPU support before running this".

Reply:
While we're figuring this out for all the projects, maybe a quick note in the readme and/or other guidelines we'd distritbute would work? For 3.4 I see that the link to the models has been upgraded so that's fine :-)

Reply:
I updated the README to add instructions for this. 

I'm still unclear how spaCy with GPU install should work. We can't specify that in requirements because it depends on the user's hardware. Maybe the steps should be 1. install spaCy with GPU 2. install requirements, and the requirements shouldn't include spaCy directly?

Reply:
Currently I have modified the README to be:

1. install spaCY with GPU support
2. install requirements
3. modify `project.yml` (for paths, GPU ID)

Let me know if there's a better way to do that.

Reply:
I assume the current state of the README is OK, let me know if not.